theano/tensor/tests/test_opt.py

```diff
@@ -2936,8 +2936,10 @@ def test_local_IncSubtensor_serialize():
 
     # Now test that the stack trace is copied over properly,
     # if we return the gradients. We need to use same mode as before.
+    # the optimizer may delete all instances of AdvancedIncSubtensor
     f = theano.function([i, j, t], dW, mode=mode)
-    assert check_stack_trace(f, ops_to_check=tensor.AdvancedIncSubtensor)
+    assert check_stack_trace(f, ops_to_check=tensor.AdvancedIncSubtensor,
```

Comment:
I think you also need to check for Subtensor and AvancedIncSubtensor1. If you do that, you may be able to remove "bug_print='ignore'".